### PR TITLE
SAAS-1084: Content Loader

### PR DESCRIPTION
**Pull request by MaksimMalakhovSimbirsoft**
[Запись экрана от 27.10.2022 14:45:11.webm](https://user-images.githubusercontent.com/107244329/198278602-d03930cb-886d-41fe-a574-7d14b8193b97.webm)

### CODE REVIEW

**Comment by TomasBlazek3PG** [comment]
Aren't the colors too strong?

**Review by TomasBlazek3PG — DISMISSED** [review]

**Review by fabio-silva — APPROVED** [review]

File: src/pages/Dashboard/Contacts/Contacts.tsx [hunk 1]
```diff
@@ -67,109 +69,119 @@ export const Contacts: FC = () => {
...
-            </a>
-          </p>
-          {CSContact.name ? (
-            <CustomerContact />
-          ) : (
-            <>
-              <p>{Messages.findUs}</p>
+    <>
+      {isLoading ? (
+        <></>
```
Comment (TomasBlazek3PG) [inline src/pages/Dashboard/Contacts/Contacts.tsx:74]: isLoading ? null : ...

File: src/pages/Dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -73,7 +73,7 @@ export const DashboardPage: FC = () => {
   }, [dispatch, orgId]);
 
   useEffect(() => {
-    setShowGettingStarted((!(companyName || orgId) || !inventory?.length) && !isPending);
+    setShowGettingStarted((!(companyName && orgId) || !inventory?.length) && !isPending);
```
Comment (TomasBlazek3PG) [inline src/pages/Dashboard/Dashboard.tsx:76]: Why the change?

Comment (MaksimMalakhovSimbirsoft) [reply src/pages/Dashboard/Dashboard.tsx]: @TomasBlazek3PG was trying different conditions. I've returned it

File: src/pages/Dashboard/Contacts/Contacts.tsx (hunk 1 above)
Comment (MaksimMalakhovSimbirsoft) [reply src/pages/Dashboard/Contacts/Contacts.tsx]: @TomasBlazek3PG I've removed it

**Review by TomasBlazek3PG — APPROVED** [review]
